src/Microsoft.IdentityModel.Logging/WilsonEventSource.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.IdentityModel.Logging
+{
+    /// <summary>
+    /// Event source based logger to log different events.
+    /// </summary>
+    public class WilsonEventSource : EventSource
+    {
+        private static EventLevel _logLevel;
+
```

Comment:
Might be a good idea to have: public const static EventLevel DefaultEventLevel = EventLevel.Error